docs/roles.rst

```diff
@@ -31,6 +31,8 @@ Example
    :value: 20
    :unit: slices
 
+.. _needref:
```

Comment:
Does this make sense here?

If yes, then a second ``:`` is missing at the end.

Reply:
This

> Does this make sense here?
> 
> If yes, then a second `:` is missing at the end.

Yes, it makes sense. It creates a link to that section so a second `:` is not needed.

Reply:
Argh, sure. My fault.
Was thinking on a directive and not the basic sphinx reference feature.
Sorry :)